saleor/graphql/product/fields.py

```diff
@@ -0,0 +1,13 @@
+from json import JSONDecodeError
+
+import graphene
+from graphql.error import GraphQLError
+
+
+class JSONStringWithValidation(graphene.JSONString):
```

Comment:
Should we use this field for `Page.content` field?

Reply:
If we want to have more readable error for invalid JSONString fields I guess we have to change all occurrences.

Reply:
Replacing is possible, gonna do it soon. For keep namings there is a problem with generating schema about different types with the same names but I will try to replace it somehow :)

Reply:
I think we already use a custom scalar for built-in floats?